package.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "@onflow/flow-nft",
+  "version": "1.0.0",
+  "description": "standard implementation of the non fungible token on flow blockchain",
+  "main": "index.js",
+  "directories": {
+    "doc": "docs",
+    "lib": "lib"
+  },
+  "files": ["contracts/*", "transactions/*"],
+  "scripts": {
+    "release": "npm run changeset version && npm run changeset publish"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/onflow/flow-nft.git"
+  },
+  "keywords": [
+    "flow-nft",
+    "nft",
+    "NonFungibleToken",
+    "flow"
+  ],
+  "author": "onFlow",
+  "license": "MIT",
```

Comment:
This license is referring to the npm package or to the nft repo? just because the flow-nft is unlicensed rather than under MIT